### ISSUE TITLE
Table error

### ISSUE THREAD

**Issue by VK-VZ** [UnamSanctam/UnamWebPanel#315]
Hello.  Today I suddenly got this error.  I did not do anything.  What could be causing this?
![Screenshot_20240107_202521_Chrome](https://github.com/UnamSanctam/UnamWebPanel/assets/149912866/27037609-edec-4fb2-97dc-b7e7b8c03980)

**Comment by UnamSanctam** [comment]
There is probably something weird in your database (`db/unamwebpanel.db`), you can check the `UNAM_LIB\Logs` file to see if there are any errors.

**Comment by VK-VZ** [comment]
https://transfiles.ru/l1ifu
Look, please, otherwise I don’t understand anything there.

**Comment by UnamSanctam** [comment]
That one is due to something else, so there's probably something weird inside the database file which causes it.

**Comment by VK-VZ** [comment]
Shall I send you a base?

**Comment by UnamSanctam** [comment]
You can send me the database file (unamwebpanel.db), if you don't want to send it here (since it contains all your miner information) then you can send it to unams@protonmail.com.

**Comment by masterjek** [comment]
I also received this error today. I deleted the old database and just copied a new empty one and everything works fine now.

**Comment by VK-VZ** [comment]
Sent

**Comment by VK-VZ** [comment]
> Я также получил эту ошибку сегодня. Я удалил старую базу данных и просто скопировал новую пустую, и теперь все работает нормально.

So the old workers will leave?  I don't want the old workers to disappear

**Comment by masterjek** [comment]
No, all the old workers will connect again. I just loaded an empty database, created a configuration for the miners in it and they all connect back.

**Comment by VK-VZ** [comment]
> You can send me the database file (unamwebpanel.db), if you don't want to send it here (since it contains all your miner information) then you can send it to [unams@protonmail.com](mailto:unams@protonmail.com).

did you look?

**Comment by DerXanRam** [comment]
> No, all the old workers will connect again. I just loaded an empty database, created a configuration for the miners in it and they all connect back.

I face this problem few weeks a go and it happened when there is a network configuration problem either from client or from server.
